packages/tools/tests/test/selenium/safari.test.ts

```diff
@@ -0,0 +1,111 @@
+import { CheckTestSuccessStatus, getGlobalConfig, LoadPlayground } from "@tools/test-tools";
+import { Builder, Button, By, Key } from "selenium-webdriver";
+
+jest.setTimeout(60000);
+
+describe("safari", () => {
+    const driver = new Builder().forBrowser("safari").build();
+
+    beforeAll(async () => {
+        await driver.get(getGlobalConfig().baseUrl + `/empty.html`);
+    });
+
+    afterAll(async () => {
+        await driver.quit();
+    });
+
+    // Check if allowMouse logic for camera touch input is validating correctly
+    // PG: https://playground.babylonjs.com/#ITQ2NZ#9
+    it("check isMouseEvent", async () => {
+        await LoadPlayground(driver, "#ITQ2NZ#9", getGlobalConfig(), 1000);
+        const el = await driver.findElement(By.id("babylon-canvas"));
+
+        // With allowMouse = true, touch controls should move camera forward if isMouseEvent is true
+        await driver.actions({ bridge: true }).move({ origin: el }).press().move({ x: 0, y: -200, origin: el }).release().perform();
```

Comment:
Did you check if `{ bridge: true }` is actually necessary like we discussed offline?

Reply:
So I looked into the bridge bool and from what I've got.  With the web-driver, if `bridge = true`, it will attempt to use the W3C protocol for handling the input automation.  This includes taking all of the input function calls and putting them into a single call, reducing latency.  If this gets rejected, it would default to the legacy actions and call each action using a legacy protocol.  Furthermore, the W3C protocol works from the middle of a WebElement, as opposed to the top-left corner like the legacy one.  Setting the bridge bool to true also adds a step to legacy calls to work from the middle, if it ends up using it.  TBH, it isn't necessary but it's convenient just for that last point.

Ref: https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/lib/input_exports_Actions.html